### PR TITLE
Fix grammars for bracket-pair-colorization

### PR DESCRIPTION
**Pull request by yamachu**
The issue was fixed in TypeScriptReact, but the problem still occurred in PlainTypeScript.
Also, VSCode did not add fields for JavaScriptReact, so I removed them from this package.

I have rewritten `~/.vscode-insiders/extensions/ms-vscode.vscode-typescript-next-4.8.20220718/package.json` to confirm that it has been corrected.

ref: https://github.com/microsoft/vscode/blob/a74adedba8fb56d14f01f7950340dab33cfeae57/extensions/typescript-basics/package.json#L63-L109
https://github.com/microsoft/vscode/blob/a74adedba8fb56d14f01f7950340dab33cfeae57/extensions/javascript/package.json#L59-L98

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,12 +48,6 @@
         "language": "javascriptreact",
         "scopeName": "source.js.jsx",
         "path": "./syntaxes/JavaScriptReact.tmLanguage.json",
-        "unbalancedBracketScopes": [
-          "keyword.operator.relational",
-          "storage.type.function.arrow",
-          "keyword.operator.bitwise.shift",
-          "punctuation.definition.tag"
-        ],
         "embeddedLanguages": {
```
Comment (hediet) [inline package.json:51]: Why did you remove it here?

**Review by hediet — CHANGES_REQUESTED** [review]

**Comment by hediet** [comment]
>  VSCode did not add fields for JavaScriptReact,

I think this is an oversight in VS Code.

**Comment by yamachu** [comment]
Thanks for reviewing!

> I think this is an oversight in VS Code.

I see, I'll revert it.

**Comment by mjbvz** [comment]
Thanks!
